src/main/java/seedu/trackermon/model/show/Comment.java

```diff
@@ -10,11 +17,19 @@
      * @param comment A valid comment.
      */
     public Comment(String comment) {
+        checkArgument(isValidComment(comment), MESSAGE_CONSTRAINTS);
         String emojiFilter = "[(\\u00a9|\\u00ae|[\\u2000-\\u3300]|\\ud83c[\\ud000-\\udfff]|"
-                + "\\ud83d[\\ud000-\\udfff]|\\ud83e[\\ud000-\\udfff])]";
+                        + "\\ud83d[\\ud000-\\udfff]|\\ud83e[\\ud000-\\udfff])]";
```

Comment:
Also what does this regex match?

Reply:
I am not sure what you mean?

Reply:
Wait never mind, I must've seen wrongly. I thought you added a new regex.